modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/AnsibleScriptConfig.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.servicetemplate;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import java.io.Serial;
+import java.io.Serializable;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * The config script of ansible.
+ */
+@Valid
+@Data
+@Slf4j
+public class AnsibleScriptConfig implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    @NotNull
+    @NotBlank
+    @NotEmpty
+    @Schema(description = "Name of the file in the script.")
```

Comment:
name of the ansible playbook. Should be the fully qualified file name (including the directory path) in the repo. Eg.,'playbook-name.yml'